slave-base/Dockerfile.rhel7

```diff
@@ -11,12 +11,16 @@ LABEL com.redhat.component="jenkins-slave-base-rhel7-container" \
       io.k8s.description="The jenkins slave base image is intended to be built on top of, to add your own tools that your jenkins job needs. The slave base image includes all the jenkins logic to operate as a slave, so users just have to yum install any additional packages their specific jenkins job will need" \
       io.openshift.tags="openshift,jenkins,slave"
 
-ENV HOME=/home/jenkins
+ENV HOME=/home/jenkins \
+    PROMPT_COMMAND="./usr/local/bin/scl_enable"
+
 
 USER root
 # Install headless Java
-RUN INSTALL_PKGS="bc gettext git java-1.8.0-openjdk-headless lsof rsync tar unzip which zip bzip2 dumb-init" && \
-    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+RUN INSTALL_PKGS="bc gettext git java-11-openjdk java-11-openjdk-headless java-1.8.0-openjdk-headless lsof rsync tar unzip which zip bzip2 dumb-init" && \
+    DISABLES="--disablerepo=rhel-server-extras --disablerepo=rhel-server --disablerepo=rhel-fast-datapath --disablerepo=rhel-server-optional --disablerepo=rhel-server-ose --disablerepo=rhel-server-rhscl" && \
+    yum $DISABLES install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    yum $DIABLES install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
```

Comment:
We shouldn't need this DISABLES, I think we are facing the issue because of dumb-init

Reply:
removed disables